File: 03 Podminky a cykly/01 funkce.ts
function zdvojnasob(n: number): number {
  return 2 * n;
}

console.log(zdvojnasob(24));

/**
 * Tahle funkce okomentuje, o jaké číslo se jedná.
 * @example
 * popisCislo(1000) // "1000 je VELKÉ číslo!"
 */
function popisCislo(n: number): string {
  if (n > 1_000) {
    return `${n} je VELKÉ číslo!`;
  }

  if (n % 2 === 0) return `${n} je sudé číslo.`;

  return `${n} je liché číslo.`;
}

console.log(popisCislo(24_002), popisCislo(2), popisCislo(115));

File: 03 Podminky a cykly/01 funkce.py
def zdvojnasob(n):
    return 2 * n

print(zdvojnasob(24))

"""
Tahle funkce okomentuje, o jaké číslo se jedná.
popisCislo(1000) // "1000 je VELKÉ číslo!"
"""
def popisCislo(n):
    if n > 1_000:
        return f"{n} je VELKÉ číslo!"

    if n % 2 == 0:
        return f"{n} je sudé číslo."
    
    return f"{n} je liché číslo."

print(popisCislo(24_002), popisCislo(2), popisCislo(115))
